docs/UserGuide.md

```diff
@@ -90,42 +90,56 @@ Shows a message explaning how to access the help page.
+* Every volunteer must have a unique `NRIC`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A volunteer can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add_volunteer n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 ag/23 vnr/S8457677H`
+* `add_volunteer n/Betsy Crowe t/graduate e/betsycrowe@example.com a/Newgate Prison p/1234567 ag/27 vnr/S8959886I t/experienced`
 
-### Listing all persons : `list`
+### Listing all persons : `list_volunteer`
 
-Shows a list of all persons in the address book.
 
-Format: `list`
+Shows a list of all volunteers in the application.
 
-### Editing a person : `edit`
 
-Edits an existing person in the address book.
+Format: `list_volunteer`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+[//]: # (### Editing a person : `edit`)
 
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tags after it.
+[//]: # ()
+[//]: # (Edits an existing person in the address book.)
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+[//]: # ()
+[//]: # (Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`)
+
+[//]: # ()
+[//]: # (* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index **must be a positive integer** 1, 2, 3, …​)
+
+[//]: # (* At least one of the optional fields must be provided.)
+
+[//]: # (* Existing values will be updated to the input values.)
+
+[//]: # (* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.)
+
+[//]: # (* You can remove all the person’s tags by typing `t/` without)
+
+[//]: # (    specifying any tags after it.)
+
+[//]: # ()
+[//]: # (Examples:)
+
+[//]: # (*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.)
+
+[//]: # (*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.)
```

Comment:
We are not implementing edits for v1.2 right? Just ask

Reply:
Yup we are not.